templates/make_manifest

```diff
@@ -7,6 +7,7 @@ STEMCELL_OS=${STEMCELL_OS:-ubuntu}
 
 infrastructure=$1
 servers=$2
+cf_manifest=$3
```

Comment:
This haproxy boshrelease is separate from the one included with CF, and designed to be a standalone system if necessary. If you want to use it in place of cf-release's haproxy, you can add the consul_agent, and metron_agent jobs from cf-release, and provide the required properties. Alternatively you could modify the haproxy properties in your CF manifest, and change the jobs for its haproxy to come from the haproxy release, instead of cf.